pkgs/data/fonts/vollkorn/default.nix

```
@@ -1,18 +1,20 @@
 { lib, stdenv, fetchzip }:
 let
   pname = "vollkorn";
-  version = "4.105";
+  majorVersion = "4";
+  minorVersion = "105";
+  fullVersion = "${majorVersion}.${minorVersion}";
 in
 fetchzip {
-  name = "${pname}-${version}";
-  url = "http://vollkorn-typeface.com/download/vollkorn-4-105.zip";
+  name = "${pname}-${fullVersion}";
+  url = "http://vollkorn-typeface.com/download/vollkorn-${majorVersion}-${minorVersion}.zip";
```

```suggestion
  url = "http://vollkorn-typeface.com/download/vollkorn-${replaceStrings ['.'] ['-'] version}.zip";
```
this can be done a bit more elegant. Not sure if it is replaceStrings or lib.replaceStrings or builtins.replaceStrings.

---

changed